cmd/ssh.go

```diff
@@ -614,7 +603,25 @@ func (cmd *SSHCmd) setupGPGAgent(
 		command = fmt.Sprintf("su -c \"%s\" '%s'", command, cmd.User)
 	}
 
-	return devssh.Run(ctx, containerClient, command, nil, writer, writer, nil)
+	cancelCtx, cancel := context.WithCancel(ctx)
+	defer cancel()
+	errChan := make(chan error, 2)
+	go func() {
+		errChan <- cmd.reverseForwardPorts(cancelCtx, containerClient, log)
```

Comment:
why do we now have a new call to `reverseForwardPorts`?

Reply:
I just moved it closer to the end of the file to indicate the two goroutines are related